kDrive/UI/Controller/RegisterViewController.swift

```diff
@@ -121,7 +121,7 @@ extension RegisterViewController: WKNavigationDelegate {
         decisionHandler: @escaping (WKNavigationActionPolicy) -> Void
     ) {
         if let host = navigationAction.request.url?.host {
-            if host == "drive.infomaniak.com" {
+            if host == "kdrive.infomaniak.com" {
```

Comment:
Can we put this in a const file ? (like we do in mail ?)

Reply:
Done